src/components/FileUpload/FileUpload.js

```diff
@@ -1,12 +1,77 @@
-//reference: https://www.positronx.io/react-file-upload-tutorial-with-node-express-and-multer/
-import React, { Component } from 'react';
-import { FormGroup } from '../Utils/Utils';
-export default class FilesUploadComponent extends Component {
-  render() {
-    return (
-            <div className="form-group">
-              <input type="file" name="uploaded_file" onChange={this.props.onChange} />
-            </div>
-    )
-  }
-}
+import React, { useState } from "react";
+import Cropper from "react-cropper";
+import "cropperjs/dist/cropper.css";
+import "./FileUpload.css";
+
+export default function FileUpload(props) {
+  const { handler, imgHandler } = props;
+
+  const [image, setImage] = useState();
+  const [saved, setSaved ] = useState("");
+  const [showCropImage, setCropImage ] = useState("hide");
+  const [cropData, setCropData] = useState("#");
+  const [cropper, setCropper] = useState();
+  const onChange = (e) => {
+    e.preventDefault();
+    let files;
+    
+    if (e.dataTransfer) {
+      files = e.dataTransfer.files;
+    } else if (e.target) {
+      files = e.target.files;
+    }
+    const reader = new FileReader();
+    reader.onload = () => {
+      setImage(reader.result);
+    };
+    reader.readAsDataURL(files[0]);
+
+    setSaved("")
+    setCropImage("")
```

Comment:
When the page is loaded the "crop section" is hidden.
![image](https://user-images.githubusercontent.com/71732016/154384752-031637d8-527b-4e50-81f8-b57956972a82.png)

![image](https://user-images.githubusercontent.com/71732016/154384780-4e00262b-55ee-41b5-bd38-732420bbe61d.png)

Reply:
Is there a way to automatically crop the image on submit if it hasn't been done yet? I walked through the process and missed the "Crop Image" button the first time around. Since I was at the bottom of the page, I didn't see the error that was updated at the top. We could do a couple of things, like putting the error right by/in the crop tool, or automatically cropping the image on submit. What do you think?

Reply:
No, there is no way to automatically crop the image by using Cropper. I moved the error message to be right above the submit button for now.
![image](https://user-images.githubusercontent.com/71732016/154771279-e83826a4-c334-4277-b495-e0902bb429f4.png)